src/network/networkd-ndisc.c

```diff
@@ -1018,6 +1018,51 @@ static int ndisc_drop_outdated(Link *link, usec_t timestamp_usec) {
         return r;
 }
 
+static int ndisc_drop_lifetime_zero_router(Link *link, sd_ndisc_router *rt) {
+        struct in6_addr router, prefix;
```

Comment:
The second commit should be redundant. Already addressed by af2aea8bb64b0dc42ecbe5549216eb567681a803.

Reply:
it did not work. The routes were still there.